include/tvm/arith/bound.h

```diff
@@ -25,7 +25,7 @@
 
 #include <tvm/arith/int_set.h>
 #include <tvm/ir/expr.h>
-#include <tvm/node/container.h>
```

Comment:
@junrushao1994 can we remove the node subdirectory now? seems to be out-dated naming now that we are in object land?

Reply:
The node subdir still contains a lot of useful stuff, like structural hash and reflection, etc. Removing node/container.h looks good to me